### ISSUE TITLE
pycbc_make_html_page is broken

### ISSUE THREAD

**Issue by duncan-brown** [ligo-cbc/pycbc#657]
This can be reproduced on sugar by running the command:
```sh
pycbc_make_html_page  --analysis-subtitle 'Week 2 (August) ER8 Chisq Tuning, 4s clustering' --analysis-title 'PyCBC Techincal Paper Run' --output-path ${HOME}/public_html/tmp/pycbc-technical/pycbc-tech-s6-august-new-chirp-bins --plots-dir /home/samantha.usman/bns/ahope-comparison-paper/pycbc-technical/s6-new-chirp-bins/august/output/results --template-file templates/orange.html
```
and you get the error:
```python
LOADING HTML FILE /home/samantha.usman/bns/ahope-comparison-paper/pycbc-technical/s6-new-chirp-bins/august/output/results//8._workflow/8.02_version/version_information_from_executables.html
Traceback (most recent call last):
  File "/home/dbrown/src/pycbc/bin/pycbc_make_html_page", line 4, in <module>
    __import__('pkg_resources').run_script('PyCBC===af539f', 'pycbc_make_html_page')
  File "/home/dbrown/src/pycbc/lib/python2.6/site-packages/pkg_resources/__init__.py", line 735, in run_script
    self.require(requires)[0].run_script(script_name, ns)
  File "/home/dbrown/src/pycbc/lib/python2.6/site-packages/pkg_resources/__init__.py", line 1652, in run_script
    exec(code, namespace, namespace)
  File "/home/dbrown/src/pycbc/lib/python2.6/site-packages/PyCBC-af539f-py2.6.egg/EGG-INFO/scripts/pycbc_make_html_page", line 226, in <module>
    output = template.render(context)
  File "/home/dbrown/src/pycbc/lib/python2.6/site-packages/jinja2/environment.py", line 969, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/dbrown/src/pycbc/lib/python2.6/site-packages/jinja2/environment.py", line 742, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/dbrown/src/pycbc/lib/python2.6/site-packages/PyCBC-af539f-py2.6.egg/pycbc/results/templates/orange.html", line 201, in top-level template code
    {{file.render()}}
  File "/home/dbrown/src/pycbc/lib/python2.6/site-packages/PyCBC-af539f-py2.6.egg/EGG-INFO/scripts/pycbc_make_html_page", line 157, in render
    return setup_template_render(self.path, self.config_path)
  File "/home/dbrown/src/pycbc/lib/python2.6/site-packages/PyCBC-af539f-py2.6.egg/pycbc/results/render.py", line 105, in setup_template_render
    output = render_default(path, cp)
  File "/home/dbrown/src/pycbc/lib/python2.6/site-packages/PyCBC-af539f-py2.6.egg/pycbc/results/render.py", line 128, in render_default
    content = fromsegmentxml(xmlfile, return_dict=True)
  File "/home/dbrown/src/pycbc/lib/python2.6/site-packages/PyCBC-af539f-py2.6.egg/pycbc/workflow/segment.py", line 694, in fromsegmentxml
    contenthandler=ContentHandler)
  File "/home/dbrown/src/pycbc/lib/python2.6/site-packages/glue/ligolw/utils/__init__.py", line 358, in load_fileobj
    ligolw.make_parser(contenthandler(xmldoc)).parse(fileobj)
  File "/usr/lib64/python2.6/xml/sax/expatreader.py", line 107, in parse
    xmlreader.IncrementalParser.parse(self, source)
  File "/usr/lib64/python2.6/xml/sax/xmlreader.py", line 123, in parse
    self.feed(buffer)
  File "/usr/lib64/python2.6/xml/sax/expatreader.py", line 207, in feed
    self._parser.Parse(data, isFinal)
  File "/usr/lib64/python2.6/xml/sax/expatreader.py", line 338, in start_element_ns
    AttributesNSImpl(newattrs, qnames))
  File "/home/dbrown/src/pycbc/lib/python2.6/site-packages/glue/ligolw/ligolw.py", line 808, in startElementNS
    raise ElementError("unknown element %s for namespace %s" % (localname, uri or NameSpace))
glue.ligolw.ligolw.ElementError: unknown element sitecatalog for namespace http://pegasus.isi.edu/schema/sitecatalog
```

**Comment by cmbiwer** [comment]
I see, the problem is that LIGOLW doesn't return a ``ValueError`` for XML files that aren't LIGOLW format so the try-except statement fails.

We need to add a case for if there is a ``glue.ligolw.ligolw.ElementError`` error.

**Comment by cmbiwer** [comment]
This should be fixed by #659. I will close this issue once #659 is merged.

**Comment by cmbiwer** [comment]
#659 has been merged. Closing issue.
